lib/server.go

```diff
@@ -34,6 +35,11 @@ func NewHTTPServer(t *Tracer, addr string) *HTTPServer {
   return s
 }
 
+func enableCors(w *http.ResponseWriter) {
```

Comment:
`http.ResponseWriter` is an [interface](https://golang.org/pkg/net/http/#ResponseWriter), no need to pass it by reference. You can change the function to:

```go
func enableCors(w http.ResponseWriter) {
  w.Header().Set("Access-Control-Allow-Origin", "*")
  w.Header().Set("Content-Type", "text/event-stream")
}

```